x-pack/plugins/security_solution/scripts/endpoint/README.md

```diff
@@ -27,17 +27,15 @@ Options:
   --seed, -s                     random seed to use for document generator
                                                                         [string]
   --node, -n                     elasticsearch node url
-                                     [string] [default: "http://localhost:9200"]
-  --alertIndex, --ai             index to store alerts in
-                                         [string] [default: "events-endpoint-1"]
+                    [string] [default: "http://elastic:changeme@localhost:9200"]
+  --kibana, -k                   kibana url
+                    [string] [default: "http://elastic:changeme@localhost:5601"]
   --eventIndex, --ei             index to store events in
                                          [string] [default: "events-endpoint-1"]
   --metadataIndex, --mi          index to store host metadata in
-                                [string] [default: "metrics-endpoint-default-1"]
+                       [string] [default: "metrics-endpoint.metadata-default-1"]
   --policyIndex, --pi            index to store host policy in
                          [string] [default: "metrics-endpoint.policy-default-1"]
-  --auth                         elasticsearch username and password, separated
```

Comment:
Keeping the auth credentials configurable may come in handy

Reply:
It's configurable in the URL options so if you need to connect to an es instances without auth you just do `http://localhost:9200`.